transifex/openformats
# Fix docx compilation error

Related to [TX-14233](https://transifex.atlassian.net/browse/TX-14233).

Problem and/or solution
-----------------------

DOCX file fails to be exported due to compilation errors caused by empty tx tags.

There was an error raised when parent element does not exist.
This is fixed by adding a check to return if the parent is None. 

How to test
-----------

1. Create a new resource with the .docx file from the ticket.
2. Upload .xliff translation.
3. Click to download translated file.
4. It should not raise any errors.

Reviewer checklist
------------------

Code:

PR:
